rs/state_machine_tests/src/lib.rs

```diff
@@ -1335,7 +1335,7 @@ impl StateMachine {
     /// and execute a round with this payload.
     /// Note that only ingress messages submitted via `Self::submit_ingress`
     /// will be considered during payload building.
-    pub fn execute_round(&self) {
+    pub fn execute_round(&self, blockmaker_metrics: Option<BlockmakerMetrics>) {
```

Comment:
Could you please rename this function to `fn execute_round_impl` and introduce a pair of public functions:
```
pub fn execute_round(&self) // calls execute_round_impl(None)
pub fn execute_round_with_blockmaker_metrics(&self, blockmaker_metrics: BlockmakerMetrics)
```
this way, there's no breaking change to the existing function